src/lib/components/Labeled.svelte

```diff
@@ -3,9 +3,14 @@
   export let label: string | null = null;
   export let explanation: string | null = null;
   export let error: string | string[] | undefined = undefined;
+  export let fullWidth: boolean = false;
 </script>
 
-<div class="relative inline-flex flex-col items-stretch">
+<div
+  class={`relative inline-flex flex-col items-stretch ${
+    fullWidth ? "w-full md:w-auto" : ""
+  }`}
+>
```

Comment:
Is this change needed? When I tested your changes it looked just as good without it.

Reply:
On small screens, the Program and Year fields looked odd to me because the rest of the elements take the full width of the screen. But it's up to your discretion and I'm happy to remove it if you think it looks good.

Reply:
You're right, I must have missed that!